sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -551,6 +576,49 @@ export class OfferingValidationModel {
     ),
   })
   yearOfStudy: number;
+  /**
+   * Indicates if the offering is an aviation offering.
+   */
+  @IsEnum(AviationYesNoOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.isAviationOffering,
+      AviationYesNoOptions,
+    ),
+  })
+  @ValidateIf((offering: OfferingValidationModel) => !!offering.programContext)
+  @ProgramIsAviationProgram(userFriendlyNames.isAviationOffering, {
+    context: ValidationContext.CreateWarning(
+      OfferingValidationWarnings.ProgramNotAviation,
+    ),
+  })
+  isAviationOffering: AviationYesNoOptions;
+  /**
+   * Indicates the aviation credential for the offering.
+   */
+  @IsEnum(AviationCredentialTypeOptions, {
+    message: getEnumFormatMessage(
+      userFriendlyNames.aviationCredentialType,
+      AviationCredentialTypeOptions,
+    ),
+  })
+  @ValidateIf(
+    (offering: OfferingValidationModel) =>
+      offering.isAviationOffering === AviationYesNoOptions.Yes,
+  )
+  @IsIn(aviationCredentialTypesEligibleForFunding, {
+    context: ValidationContext.CreateWarning(
+      OfferingValidationWarnings.AviationCredIsPrivatePilotTraining,
+    ),
+  })
```

Comment:
Can we add a comment here to explain the context here? I took a while for me to get the background.

Reply:
The comment doesn't seem to indicate the validation happening here. Can you please adjust?